mplc/multi_partner_learning.py

```diff
@@ -109,7 +109,8 @@ def compute_test_score_for_single_partner(self, partner):
 
         # Save model score on test data
         self.test_score = model_evaluation_test_data[1]  # 0 is for the loss
-        self.nb_epochs_done = (es.stopped_epoch + 1) if es.stopped_epoch != 0 else self.epoch_count
+        self.loss_collective_models.append(model_evaluation_test_data[0])  # store the loss for PVRL
+        self.nb_epochs_done = (es.stopped_epoch + 1) if self.is_early_stopping else self.epoch_count
```

Comment:
Looking at Eric's comment here --> https://github.com/SubstraFoundation/distributed-learning-contributivity/pull/141#discussion_r428158162
I think this won't work.

Reply:
Actually this line was not working when the earlystooping was deactivated because the `es` object was no more instantiated in that case. This new line also to fix this.

Reply:
Ok I understand.
However I think Eric's [comment](https://github.com/SubstraFoundation/distributed-learning-contributivity/pull/141#discussion_r428158162) still holds: 

> Via the callback the EarlyStopping provides stopped_epoch. But the way it is implemented, if early stopping wasn't trigger, this returns 0. 


So if `is_early_stopping = true` but the early stopping was never triggered, `stopped_epoch = 0`